docs/conf.py

```diff
@@ -85,7 +86,11 @@
 
 html_theme = 'sphinx_rtd_theme'
 html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
-html_static_path = []
+html_static_path = ['_static']
```

Comment:
Why is this needed? IIRC docs build fine without adding the `_static` but adding this results in a warning.